.env.example

```
@@ -23,6 +23,10 @@ ENVIRONMENT=development
 # Server Configuration
 SERVER_PORT=8080
 SERVER_HOST=0.0.0.0
+# Comma-separated list of CORS allowed origins. Leave unset to default to http://localhost:3000.
+# In production, set to the explicit origins that should be able to call the API (e.g. https://app.example.com,https://admin.example.com).
+# Setting this to "*" is NOT recommended and is no longer the default for security reasons.
+NEURATRADE_CORS_ALLOWED_ORIGINS=https://app.example.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEURATRADE_CORS_ALLOWED_ORIGINS key should go before the SERVER_HOST key

(UnorderedKey)